File: fontes/c/micro/micro02.c
#include<stdio.h>

int main()
{
 int num1;
 int num2;

 printf("Digite o primeiro número: ");
 scanf("%d", &num1);
 printf("Digite o segundo número: ");
 scanf("%d", &num2);
 if (num1 > num2)
 {
 	printf("O primeiro número %d é maior que o segundo %d", num1,num2);
 }
 else
 {
 	printf("O segundo número %d é maior que o primeiro %d", num2, num1);
 }
 return 0;
}

File: fontes/pascal/micro/micro02.pas
program Micro02;
    
var 
	num1, num2: INTEGER;
begin
    writeln('Digite o primeiro numero: ');
    readln(num1);
    writeln('Digite o segundo numero: ');
    readln(num2);
    if num1>num2 then
        writeln('O primeiro numero ',num1,' eh maior que o segundo numero ',num2)
    else
        writeln('O segundo numero ',num2,' eh maior que o primeiro numero ',num1);
end.
